packages/website/lib/api.ts

```diff
@@ -141,28 +141,35 @@ export async function getSingleArticleBySlug(slug, preview = false) {
 }
 
 export async function getAllArticles(preview = false) {
-  const entries = await fetchGraphQL(
-    `query {
-      kbAppArticleCollection(where: { slug_exists: true }) {
-        items {
-          ${ARTICLE_GRAPHQL_FIELDS}
+  try {
```

Comment:
why do you think it's better to just mute these errors?

Reply:
Oops, that file was not supposed to go in. But it lets you work on the docs website when you're offline 😎